ambari-logsearch-config-json/src/main/java/org/apache/ambari/logsearch/config/json/model/inputconfig/impl/InputConfigImpl.java

```diff
@@ -39,15 +39,17 @@
   @ShipperConfigElementDescription(
     path = "/input",
     type = "list of json objects",
-    description = "A list of input descriptions"
+    description = "A list of input descriptions",
+    examples = {"{\"input\" : [ {\"type\": \"myinput_service_type\"}] }"}
```

Comment:
What do you think? Would adding the following is helpful:
```
where myinput_service_type is like ambari_server, hdfs_namenode etc...
```

Reply:
@kasakrisz ok, could be